.go-version

```diff
@@ -0,0 +1 @@
+1.16.3
```

Comment:
Can we bump this to the current [1.18](https://tip.golang.org/doc/go1.18)?
```suggestion
1.18.3
```

Reply:
the old provider framework had issues with G0 1.17.
that's why we are at 1.16